test/unit/utility/views/slice_test.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include <seqan3/test/expect_range_eq.hpp>
 #include <seqan3/test/expect_same_type.hpp>
+#include <seqan3/utility/range/concept.hpp>
```

Comment:
I  assume that this inlcude was previously given from somewhere else.

Reply:
Yup, `slice.hpp` included `take_view.hpp` which included `concept.hpp`. Now that `take_view.hpp`, we have to add it here